fw/thread.go

```diff
@@ -28,21 +28,21 @@ var Threads map[int]*Thread
 var LOCALHOST = []byte{0x6c, 0x6f, 0x63, 0x61, 0x6c, 0x68, 0x6f, 0x73, 0x74}
 
 // HashNameToFwThread hashes an NDN name to a forwarding thread.
-func HashNameToFwThread(name enc.Name) int {
+func HashNameToFwThread(name *enc.Name) int {
```

Comment:
`enc.Name` is already a slice:

https://github.com/zjkmxy/go-ndn/blob/ec49b726e8ddecf63698832b53f01f2ec81aa72b/pkg/encoding/name_pattern.go#L12